api/AnimationTimeline.json

```diff
@@ -13,55 +13,19 @@
           "edge": {
             "version_added": "≤79"
           },
-          "firefox": [
-            {
-              "version_added": "75"
-            },
-            {
-              "version_added": "63",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "dom.animations-api.timelines.enabled",
-                  "value_to_set": "true"
-                }
-              ]
-            },
-            {
-              "version_added": "48",
-              "version_removed": "63",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "dom.animations-api.core.enabled",
-                  "value_to_set": "true"
-                }
-              ]
-            }
-          ],
-          "firefox_android": [
-            {
-              "version_added": "63",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "dom.animations-api.timelines.enabled",
-                  "value_to_set": "true"
-                }
-              ]
-            },
-            {
-              "version_added": "48",
-              "version_removed": "63",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "dom.animations-api.core.enabled",
-                  "value_to_set": "true"
-                }
-              ]
-            }
-          ],
+          "firefox": {
+            "version_added": "75"
+          },
+          "firefox_android": {
+            "version_added": "63",
+            "flags": [
```

Comment:
Is the existing data just wrong here? It seems unlikely this was shipped in desktop but left behind a flag on Android.

Reply:
I could answer my own question and sent https://github.com/mdn/browser-compat-data/pull/9452.